src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -311,6 +305,44 @@ internal TAttribute ResolveFromBindings(IReadOnlyDictionary<string, object> bind
             return newAttr;
         }
 
+        // Resolution policy for { } in  binding templates. 
+        // The default policy is just a direct substitution for the binding data. 
+        // Derived policies can enforce formatting / escaping when they do injection. 
+        private class ResolutionPolicy
+        {
+            public virtual string TemplateBind(BindingTemplate template, IReadOnlyDictionary<string, object> bindingData)
+            {
+                if (bindingData == null)
+                {
+                    return template.Pattern;
+                }
+                return template.Bind(bindingData);
+            }
+
+            // Expand on [AutoResolve] property to make this pluggable, 
+            public static ResolutionPolicy GetPolicy(PropertyInfo propInfo)
+            {
+                if (propInfo.DeclaringType == typeof(TableAttribute) && propInfo.Name == "Filter")
```

Comment:
In this PR I think we should just put an attribute in place allowing this to be specified rather than be hardcoded like this.